config/routes.rb

```diff
@@ -14,6 +14,7 @@
   get '/dashboard', to: 'users#show'
   get '/travel_summary', to: 'journey#summary'
   get '/travel_game', to: 'journey#game'
+  get '/help', to: 'journey#help'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.